organize/models.py

```diff
@@ -109,9 +124,20 @@ def deploy(self):
         # copy old one or copy old and change organizaers.
         event = self.create_event()
 
-        # TODO: use method created in separate branch to create gmail acconut
-        # and get password from it.
-        password = "FAKE_PASS"
+        # sort out Gmail accounts
```

Comment:
[In a perfect world 😎] I would move the logic of the following few lines to `get_or_create_gmail` or something like this, and let it return the `(email, email_password|None)` tuple.